galleries/examples/spines/spines_dropped.py

```diff
@@ -3,7 +3,7 @@
 Dropped spines
 ==============
 
-Demo of spines offset from the axes (a.k.a. "dropped spines").
+Demo of spines offset from the Axes (a.k.a. "dropped spines").
```

Comment:
This one I'm not quite sure we want to evoke the `Axes` object here, as the spines are _part_ of the `Axes` object, and they are not offset from such. I could go either way, but feels slightly different.